workspaces/linux-host/Makefile

```diff
@@ -0,0 +1,66 @@
+# Makefile
+#
+# AUTHORS
+#
+# The Veracruz Development Team.
+#
+# COPYRIGHT AND LICENSING
+#
+# See the `LICENSING.markdown` file in the Veracruz root directory for
+# licensing and copyright information.
+
+.PHONY: all doc clean fmt
+
+PROXY_ATTESTATION_SERVER_DB  = $(abspath proxy-attestation-server.db)
+TEST_PARAMETERS = DATABASE_URL=$(PROXY_ATTESTATION_SERVER_DB) \
+	VERACRUZ_POLICY_DIR=$(abspath ../linux-runtime/test-programs) \
+	VERACRUZ_TRUST_DIR=$(abspath crates/test-collateral) \
+	VERACRUZ_PROGRAM_DIR=$(abspath ../linux-runtime/test-programs) \
+	VERACRUZ_DATA_DIR=$(abspath crates/sdk/datasets) \
+	LINUX_ROOT_ENCLAVE_PATH=$(abspath ../linux-runtime/target/release/linux-root-enclave) \
+	RUNTIME_MANAGER_ENCLAVE_PATH=$(abspath ../linux-runtime/target/release/runtime_manager_enclave)
+CC = CC_x86_64_unknown_linux_gnu=gcc CC_aarch64_unknown_linux_gnu=gcc
+
+all:
+	$(CC) cargo build --release \
+		--features proxy-attestation-server/linux \
+		--features veracruz-client/linux \
+		--features veracruz-server/linux \
+		--features veracruz-server-test/linux \
+		--features veracruz-test/linux \
+		--features cli
+
+datasets:
+	$(MAKE) -C ../host datasets
+
+policy-files:
+	$(MAKE) -C ../linux-runtime policy-files
+
+test-server: datasets policy-files $(PROXY_ATTESTATION_SERVER_DB)
+	$(CC) $(TEST_PARAMETERS) cargo test -p veracruz-server-test --features linux --no-run -- --nocapture
```

Comment:
Why are we doing `--no-run` first? Doesn't this just increase the build time with no benefit?

Reply:
This is copied from the top-level Makefile (https://github.com/mathias-arm/veracruz/blob/pr/workspaces-initial/Makefile#L123). This might be some smoke-test that helps debug the tests.